web/js/lib/visualsearch/visualsearch.js

```diff
@@ -156,6 +156,7 @@ VS.ui.SearchBox = Backbone.View.extend({
   // Used to launch a search. Hitting enter or clicking the search button.
   searchEvent : function(e) {
     var query = this.value();
+    console.log(query);
```

Comment:
It's good practice to remove these console logs that you put in while debugging. 👍